### ISSUE TITLE
Do you guys plan to make user data tie to user and not to the containers 

### ISSUE THREAD

**Issue by abhishekms1047** [Vectordash/vectordash-cli#3]
Do you guys plan to make user data tie to user and not to the containers .

For example :-- If user uploads training data into machine it should not be tied up to that container, but it should be moved into a -ever persistent machine . 
Whenever the user shuts down a machine all the details have to be saved to that ever persistent machine . 
As he launches new machine(container) ,it should pull the users folders from ever persistent machine  and set the environment with user data(that he has used in his earlier instance) .

*This would enable the user to carry his experiments across different instances.
*From cost perspective ,this would require you guys to host a ever persistent machine to keep all the data-(S3 might be a solution)

**Comment by Samin100** [comment]
Yes! This is a feature we're planning on launching within the next 7 days. How large are the datasets you typically work with?
